cli/cmd/sync.go

```diff
@@ -281,3 +297,14 @@ func sync(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func pluginEnv(environ []string, pluginName, kind string) []string {
+	env := make([]string, 0)
+	prefix := "__" + kind + "_" + pluginName + "__"
```

Comment:
This elaborate prefixing scheme is used to prevent things like a plugin being named `OTEL` and then getting access to OTEL environment variables.